service/db/src/main/java/gov/va/vro/service/db/SaveToDbServiceImpl.java

```diff
@@ -105,6 +172,95 @@ public void insertEvidenceSummaryDocument(GeneratePdfPayload request, String doc
     claimRepository.save(claim);
   }
 
+  @Override
+  @Transactional
+  public void insertOrUpdateExamOrderingStatus(ExamOrder examOrder) {
+    ExamOrderEntity examOrderEntity =
+        examOrderRepository
+            .findByCollectionId(examOrder.getCollectionId())
+            .orElseGet(() -> createExamOrder(examOrder));
+    if (null != examOrderEntity) {
+      examOrderEntity.setCollectionId(examOrder.getCollectionId());
+      examOrderEntity.setStatus(examOrder.getStatus());
+      examOrderRepository.save(examOrderEntity);
+    }
+  }
+
+  @Override
+  public void insertFlashIds(List<String> veteranFlashIds, String veteranIcn) {
+    var veteran = veteranRepository.findByIcn(veteranIcn);
+    if (veteran.isEmpty()) {
+      log.warn("Could not find a Veteran with this ICN. Could not attach flash IDs.");
+      return;
+    }
+    if (veteranFlashIds == null) {
+      log.warn("The Veteran Flash ID list was null, could not attach to Veteran.");
+      return;
+    }
+    VeteranEntity entity = veteran.get();
+    List<VeteranFlashIdEntity> flashIdList = createFlashIds(veteranFlashIds, entity);
+    entity.setFlashIds(flashIdList);
+    veteranRepository.save(entity);
+  }
+
+  @Override
+  public void updateRfdFlag(String benefitClaimId, boolean rfdFlag) {
+    var claim = claimRepository.findByVbmsId(benefitClaimId);
+    if (claim.isEmpty()) {
+      log.warn("Could not find claim with id and idType, could not update RFD flag.");
+      return;
+    }
+    ClaimEntity claimEntity = claim.get();
+    claimEntity.setRfdFlag(rfdFlag);
+    claimRepository.save(claimEntity);
+  }
+
+  @Override
+  public void updateSufficientEvidenceFlag(
+      String claimSubmissionId, Boolean flag, String diagnosticCode) {
+    // ClaimSubmissionId in v1 = ReferenceId on ClaimSubmission in VRO
+    Optional<ClaimSubmissionEntity> claimSubmission =
+        claimSubmissionRepository.findFirstByReferenceIdAndIdTypeOrderByCreatedAtDesc(
+            claimSubmissionId, Claim.DEFAULT_ID_TYPE);
```

Comment:
Somehow, you'll need to get the claim or `idType`.
```suggestion
            claimSubmissionId, claim.getIdType());
```